src/AVFoundation/AVCaptureReactionType.rgen.cs

```diff
@@ -0,0 +1,44 @@
+#pragma warning disable APL0003
```

Comment:
What's this warning? I didn't see it in a search for it.

Reply:
It is defined via a ExperimentalAttribute: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-12.0/experimental-attribute 

This was added in a previous PR, which is documented under https://github.com/xamarin/xamarin-macios/blob/main/docs/preview-apis.md#rgen-apl0003 i

Since we are using them, we have to ensure that we do not have a compilation warning.